lte/protos/sctpd.proto

```diff
@@ -25,7 +25,9 @@ message InitReq {
     repeated string ipv6_addrs = 4; // ipv6 addresses to listen on
```

Comment:
because you changed proto, do you need to create "go" files and commit those also? if not, does the jenkins job go well?